release/release.go

```diff
@@ -37,41 +37,33 @@ type OS struct {
 	VersionID string `json:"version-id,omitempty"`
 }
 
+var (
+	versionSignaturePath = "/proc/version_signature"
```

Comment:
That's what I'm saying. 'on ubuntu with a !ubuntu kernel' *will* degrade here, since the !ubuntu kernel won't have /proc/version_signature. That !ubuntu kernel may have the Ubuntu patches-- we just don't know.